### ISSUE TITLE
Need to reload the page on initial form setup

### ISSUE THREAD

**Issue by vandebo** [aprotim/cordially#2]
After creating the "Invitees" sheet and setting the appropriate header columns (per the help text), the page needs to be reloaded before the next step is possible.  This step should be added to the instructions.

**Comment by aprotim** [comment]
I don't think this is necessary in general? My experience is that this happens if you've opened the spreadsheet before you've installed the add-on and Docs just not handling that race correctly. Can you check if you open a new spreadsheet if the full Cordially menu is available now?

**Comment by vandebo** [comment]
A new sheet works correctly.  A note in the help menu wouldn't hurt.
